### PR TITLE
Merge two identical dtags & add Attributes heading

### PR DESCRIPTION
**Pull request by IanMadd**
Signed-off-by: IanMadd <imaddaus@chef.io>

### Description

This replaces the dtag `resources_common_guards_properties` with `resources_common_guards_attributes`. The text of these two dtags was identical, the only difference was that the `..._attributes` dtag was preceded by the heading **Attributes** and the `..._properties` dtag was not. 

This also adds the **Attributes** heading if it was not there, and replaces the word "properties" with "attributes" in the following paragraph.

### Check List

- [ ] Spell Check
- [X] Local build
- [X] Examine the local build
- [ ] All tests pass

### CODE REVIEW

**Comment by mjingle** [comment]
Great work! I believe that these modified resource files may affect our auto-generating resources doc project. Going to tag @tas50 here just in case.

File: chef_master/source/resource_apt_package.rst [hunk 1]
```diff
@@ -261,9 +261,11 @@ A guard property is useful for ensuring that a resource is idempotent by allowin
 
 .. end_tag
 
-.. tag resources_common_guards_properties
+**Attributes**
```
Comment (tas50) [inline chef_master/source/resource_apt_package.rst]: These should be called Properties

File: chef_master/source/resource_common.rst [hunk 2]
```diff
@@ -135,7 +135,7 @@ A guard property is useful for ensuring that a resource is idempotent by allowin
 
 Attributes
 -----------------------------------------------------
-.. tag resources_common_guards_attributes
+.. tag resources_common_guards_properties
```
Comment (mjingle) [inline chef_master/source/resource_common.rst:138]: The title above needs to be changed then to Properties.

**Review by mjingle — APPROVED** [review]
tenor-212051677

**Pull request merged**